sds_data_manager/constructs/backup_bucket_construct.py

```diff
@@ -14,15 +14,7 @@ class BackupBucket(Construct):
     should be the AWS account for the source bucket.
 
     For replication to work, you also need to deploy SdsDataManager and create
```

Comment:
Can you replace `SdsDataManager` with `SDCStack`? It must be remains from a while ago.
```suggestion
    For replication to work, you also need to deploy SDCStack and create
```